src/MangaJet/app/src/main/java/com/mangajet/mangajet/ui/search/SearchFragment.kt

```diff
@@ -96,27 +120,59 @@ class SearchFragment : Fragment() {
         searchViewModel =
             ViewModelProvider(this).get(SearchViewModel::class.java)
 
+        // init binding
         _binding = SearchFragmentBinding.inflate(inflater, container, false)
         val root: View = binding.root
 
+        // init toolbar
         mToolbar = binding.searchToolbar
         mToolbar?.inflateMenu(R.menu.search_menu)
         mToolbar?.setOnMenuItemClickListener{
             onOptionsItemSelected(it)
         }
 
+        // init result layout
         binding.noResultLayout.visibility = View.VISIBLE
         binding.progressBar.hide()
 
+        // init list with mangas
         var searchView = binding.searchView
-
         activity?.let {
             searchView.setOnQueryTextListener(OnQueryTextListener(it))
         }
 
         return root
     }
 
+
+    override fun onStart() {
+        super.onStart()
+
+        // init mangas (tag search if we need this)
+        activity?.let {
+            if (MangaJetApp.isNeedToTagSearch) {
+                MangaJetApp.isNeedToTagSearch = false
+                binding.searchView.setQuery("tag := " + MangaJetApp.tagSearchInfo!!.first,
```

Comment:
Good idea to put this "tag := " to Librarian.settings, because it is magic constant...